react/components/ProductPrice/Installments.js

```diff
@@ -45,53 +78,35 @@ export default class Installments extends Component {
     const [installmentsElement, installmentPriceElement, timesElement] = [
       installment.NumberOfInstallments,
       formattedInstallmentPrice,
-      <span key="times">&times;</span>,
+      <Fragment>&times;</Fragment>,
     ].map((element, index) => (
-      <span className="vtex-price-installments__value" key={index}>
+      <span className={classes.installmentValue} key={index}>
         {element}
       </span>
     ))
 
     return (
-      <div className="vtex-price-installments__container c-muted-2">
-        <div className="vtex-price-installments dib">
-          {showLabels ? (
-            <FormattedMessage
-              id="pricing.installment-display"
-              values={{
-                installments: installmentsElement,
-                installmentPrice: installmentPriceElement,
-                times: timesElement,
-              }}
-            />
-          ) : (
-            <span>
+      <div className={classes.root}>
```

Comment:
But thats the thing, if you're passing classes to the elements do we need these app tokens at all? Because the only utilify for them would be styling the elements writting css and we are trying to avoid this. So it seems to me that if the user needs to style the elements via css he is going to pass his own class and style it, maybe should have something in docs saying that and advising to pass a class named `vtex-price-installments`, this way would make the component more cleaner.

Reply:
I still think they are necessary, we don't want to limit the capacity to which the user can customize this app

Reply:
yeah. still make senses.

Reply:
But how is this limiting the user if you can still pass the token with something like `classes.installments = 'vtex-price__installments ph1 dib flex ...'`?